assets/src/js/admin/reports/components/ProductRecommendation/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {useState} from 'react';
+import {__} from '@wordpress/i18n';
+import {createInterpolateElement} from '@wordpress/element';
+import {getWindowData} from '../../utils';
+import './style.scss';
+import {
+    RecommendedProductData,
+    useRecommendations,
+} from '../../../../../../../src/Promotions/sharedResources/hooks/useRecommendations';
```

Comment:
You can use `@givewp/promotions/sharedResources/hooks/useRecommendations`

Reply:
Yea, I was originally worried it might not work as expected outside of my local env. I get a module warning with a red underline when I add it. Il double check and make the switch. Thanks!